docker-compose.yml

```diff
@@ -1,12 +1,22 @@
 version: '3'
 services:
+
     web:
-        build: ./
+        build: ./.docker/web
         container_name: svgtranslate_web
         ports:
           - ${TOOLFORGE_DOCKER_PORT}:80
         volumes:
-          - ./.docker/lighttpd.conf:/etc/lighttpd/lighttpd.conf:cached
+          - ./.docker/web/lighttpd.conf:/etc/lighttpd/lighttpd.conf:cached
           - ./:/var/www:cached
         environment:
           TOOLFORGE_DOCKER_PORT: ${TOOLFORGE_DOCKER_PORT}
+
+    assets:
+        build:
+            dockerfile: ./.docker/assets/Dockerfile
+            context: .
+        container_name: svgtranslate_assets
+        volumes:
+          - ./:/svgtranslate:cached
+        tty: true
```

Comment:
why do you need this?

Reply:
I wanted to keep the container running, so that the developer could connect to it and run webpack etc. as required. I'm guessing I'm on the wrong track? :)